client/verta/verta/_registry/modelversion.py

```diff
@@ -184,6 +201,23 @@ def log_artifact(self, key, asset, overwrite=False):
         self._upload_artifact(key, artifact_stream, artifact_type=artifact_type)
 
     def get_artifact(self, key):
+        """
+        Gets the artifact with name `key` from this Model Version.
+
+        If the artifact was originally logged as just a filesystem path, that path will be returned.
+        Otherwise, bytes representing the artifact object will be returned.
+
+        Parameters
+        ----------
+        key : str
+            Name of the artifact.
+
+        Returns
+        -------
+        str or bytes
+            Filesystem path or bytes representing the artifact.
```

Comment:
This method can return the object itself, too!
(suggestion taken from `ExperimentRun.get_artifact()`)
```suggestion
            Filesystem path of the artifact, the artifact object, or a bytestream representing the
            artifact.
```